maven_iuvs/file_classes.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, filename):
         self.__basename = _os.path.basename(filename)
         self.__check_input_is_iuvs_data_filename()
         hdulist = HDUList.fromfile(filename, mode='readonly',
-                                   memmap=True, save_backup=False,
+                                   memmap=False, save_backup=False, # Note: Changed memmap to False because having it True precluded making quicklooks. --EMC 8/10/23
```

Comment:
We could stop using this file_class object... it seriously slows down find_files and isn't super helpful for my workflow... immaterial to the present PR but interested in your thoughts.

Reply:
I've used it in a few places, but I have no huge attachment. It is nice that it has some methods to obtain certain ephemera/data easily but it's also not enormously important. I'd like to table it for now or at least introduce a new commit to remove it, as long as that wouldn't bother anyone...

Reply:
Happy to change all the loading routines to use fits.open() instead; IUVSFITS adds a lot of overhead processing time.

Reply:
Done in ced635c7bcaa29b50074b1e9d4643fb3929406f1, there weren't that many in use, mostly I had used it in echelle.py